database/engine/README.md

```diff
@@ -41,7 +41,7 @@ can select the memory mode for localhost by editing netdata.conf and setting:
 For setting the memory mode for the rest of the nodes you should look at
 [streaming](../../streaming/).
 
-The `history` configuration option is meaningless for `memory mode = dbengine` and is ignored for any metrics being
+>**Tip!** For `memory mode = dbengine`, the `history` configuration option is meaningless and is ignored for any metrics being
```

Comment:
Maybe this could be "When using `memory mode = dbengine`..."? Just an idea.

```suggestion
>**Tip!** When using `memory mode = dbengine`, the `history` configuration option is meaningless and is ignored for any metrics being
```